README.md

```diff
@@ -1508,7 +1508,10 @@ IEEE Robotics and Automation Letters (RA-L), 5(3):4596-4603, July 2020.
 - <a name="Patel21abec"></a>Patel, H., Iaboni, C., Lobo, D., Choi, J., Abichandani, P.,  
 *[Event Camera Based Real-Time Detection and Tracking of Indoor Ground Robots](https://arxiv.org/pdf/2102.11916v1)*,  
 arXiv, 2021.
-
+- <a name="Kim21nimg"></a>Kim, J., Bae, J., Park, G., Zhang, D., and Kim, Y.,  
```

Comment:
typically we use the conference in the key: e.g., Kim21iccv

Reply:
Thanks! I have changed the key to Kim21iccv.